lib/datasource/git-tags/index.ts

```diff
@@ -1,20 +1,16 @@
-import simpleGit from 'simple-git/promise';
+import { ReleaseResult, GetReleasesConfig } from '../common';
 import * as semver from '../../versioning/semver';
 import { logger } from '../../logger';
-import { ReleaseResult, GetReleasesConfig } from '../common';
+import * as gitRefs from '../git-refs';
 
 export const id = 'git-tags';
 
 const cacheNamespace = 'git-tags';
```

Comment:
I don't think we need to cache this anymore if we're caching the getRawRefs() result, right?

Reply:
yeah, that's right ! but what I thought was, still if this is cached and then we can even save the process of filtering the tags from it for that much time!, what do you think shall I remove git-refs and git-tags cache and only cache the getRawRefs() results?

Reply:
Yes, remove both cases and only cache the getRawRefs results

Reply:
okay